### PR TITLE
DOC: add advive for citing nibabel

### PR DESCRIPTION
**Pull request by matthew-brett**
Zenodo DOIs and instructions for making them.

### CODE REVIEW

**Comment by effigies** [comment]
Looks good! Thanks for doing that. I was looking for a way to cite nibabel in the last couple days.

**Comment by effigies** [comment]
(Still looks fine, if you're waiting for review.)

**Comment by matthew-brett** [comment]
:) - thanks !

File: doc/source/devel/make_release.rst [hunk 1]
```diff
@@ -292,6 +292,22 @@ Release checklist
...
+* Go to: https://github.com/nipy/nibabel/tags and select the new tag, to fill
+  in the release notes.  Copy the relevant part of the Changelog into the
+  release notes.  Click on "Publish release".  This will cause Zenodo_ to
+  generate a new release "upload", including a DOI.  After a few minutes, go
+  to https://zenodo.org/deposit and click on the new release upload.  Click on
+  the "View" button and click on the DOI badge at the right to display the
+  text for adding a DOI badge in various formats. Copy the DOI Markdown text.
+  The markdown will look something like this::
+
+    [![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.60847.svg)](http://dx.doi.org/10.5281/zenodo.60847)
```
Comment (yarikoptic) [inline doc/source/devel/make_release.rst]: great. I have also adjusted citation injection in duecredit, also to cite nibabel if imported: https://github.com/yarikoptic/duecredit/commit/f8e2c2a630c179f7f37f35d53a83f0478540cb24

I wondered now, would you mind if I submit a PR to add duecredit's "native" stub within nibabel thus avoiding necessity to inject its citation at run-time? ;)

**Comment by yarikoptic** [comment]
On Thu, 25 Aug 2016, Chris Markiewicz wrote:

>    Looks good! Thanks for doing that. I was looking for a way to cite nibabel
>    in the last couple days.

next time, look no further but use duecredit ;) e.g. by running your
analysis script via

python -m duecredit

;)
-- 
Yaroslav O. Halchenko
Center for Open Neuroscience     http://centerforopenneuroscience.org
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik

**Pull request merged**